pywhat/what.py

```diff
@@ -8,14 +50,26 @@
     )
 )
 @click.argument("text_input", required=True)
-def main(text_input):
+@click.option("--tags", is_flag=True, expose_value=False, callback=print_tags, help="Show available tags and exit.")
+@click.option("--rarity", help="Filter by rarity.")
+@click.option("--include_tags", help="Only print entries with included tags.")
+@click.option("--exclude_tags", help="Exclude tags.")

```

Comment:
Can we assign short names here? Just copy what I did (I don't wanna write to your branch and potentially over-write stuff you've done)

```py
@click.argument("text_input", required=True)
@click.option("-t", "--tags", is_flag=True, expose_value=False, callback=print_tags, help="Show available tags and exit.")
@click.option("-r", "--rarity", help="Filter by rarity. This is in the range of 0:1. To filter only items past 0.5, use 0.5: with the colon on the end.")
@click.option("-i", "--include_tags", help="Only print entries with included tags.")
@click.option("-e", "--exclude_tags", help="Exclude tags.")
```